src/dashboard/src/utils/oidc.ts

```diff
@@ -268,7 +272,7 @@ class OIDC {
 
   public async verifyToken(token: string) {
     // 1. Decode the ID token.
-    const { header } = jws.decode(token);
+    const { header } = jws.decode(token) as ISignatureToken;
```

Comment:
Header is no longer in the types for some reason.  Still need to test to confirm whether this will work.